src/async-generators/default/async-class-decl-method.template

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2017 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+path: language/statements/class/async-gen-method-
+name: Async Generator method as a ClassDeclaration element
+esid: prod-AsyncGeneratorMethod
+info: |
+  ClassElement :
+    MethodDefinition
+
+  MethodDefinition :
+    AsyncGeneratorMethod
+
+  Async Generator Function Definitions
+
+  AsyncGeneratorMethod :
+    async [no LineTerminator here] * PropertyName ( UniqueFormalParameters ) { AsyncGeneratorBody }
+---*/
+
+var callCount = 0;
+
+class C { async *gen() {
+    callCount += 1;
+    /*{ body }*/
+}}
```

Comment:
I'm following the pattern here: https://github.com/tc39/test262/blob/master/src/generators/default/class-method-definition.template#L22

It confused me at first, but this is a good hack to reuse the same indentation from the cases as they are using four spaces for the other function forms.

It's a trade off, the generated tests become easier to read this way.